packages/tasks/src/local-apps.ts

```diff
@@ -48,16 +48,14 @@ function isGgufModel(model: ModelData) {
 
 const snippetLlamacpp = (model: ModelData): string[] => {
 	return [
-		`
-## Install and build llama.cpp with curl support
-git clone https://github.com/ggerganov/llama.cpp.git 
-cd llama.cpp
-LLAMA_CURL=1 make
+		`## Install llama.cpp via brew or from source https://github.com/ggerganov/llama.cpp?tab=readme-ov-file#build
+
+brew install llama.cpp
```

Comment:
```suggestion
		`## Install llama.cpp via brew
brew install llama.cpp

## or from source with curl support
git clone https://github.com/ggerganov/llama.cpp
cd llama.cpp
LLAMA_CURL=1 make
```

WDYT?

Reply:
The only problem her is that 

```
git clone https://github.com/ggerganov/llama.cpp && cd llama.cpp && LLAMA_CURL=1 make  
```

would compile to metal only by default (so it'd have the same effect as brew)

People need to pass different flags for different backends

CUDA Build would require `LLAMA_CUDA=1` (ref: https://github.com/ggerganov/llama.cpp?tab=readme-ov-file#blas-build), different for BLAS, VULKAN, etc.

Maybe I'm overthink this, and the way you suggest is okay. but, thought it's good to flag this. Let me know what ya think!

Reply:
I also thought that it'd be nice to give more options. We could potentially say after the `brew` command something like "This will enable super-fast Metal inference on a Mac. For more options or to target other backends, you can install from source with curl support. For example, to compile with CUDA on Linux..."

Reply:
edited my suggestion, WDYT?

Reply:
Yeah! This looks like a good compromise to me. I think the key is to say compile with `LLAMA_CURL`.